README.md

```diff
@@ -1,2 +1,56 @@
 # Enumerables
-Recreate Ruby’s enumerable methods
+In this project we re-created some of the Ruby's enumerable methods like each, all, any, none, count, inject and map. 
+## Built With
+
+- Ruby 2.7.1
+## Getting Started
+
+- To clone enter this command in your terminal 
+- `git clone git@github.com:Haroonabdulrazaq/Enumerables.git
```

Comment:
That won't work because it is using the SSH connection protocol, use instead HTPPS.
```suggestion
- `git clone https://github.com/Haroonabdulrazaq/Enumerables.git`
```